pygmt/figure.py

```diff
@@ -75,12 +75,9 @@ class Figure:
     """
     A GMT figure to handle all plotting.
 
-    Use the plotting methods of this class to add elements to the figure.  You
-    can preview the figure using :meth:`pygmt.Figure.show` and save the figure
-    to a file using :meth:`pygmt.Figure.savefig`.
-
-    Unlike traditional GMT figures, no figure file is generated until you call
-    :meth:`pygmt.Figure.savefig` or :meth:`pygmt.Figure.psconvert`.
```

Comment:
This sentence no longer makes sense, since in GMT modern mode, no figure files are generated until `gmt end`.

For reference, these lines were added in 2017 at the start of this project, and GMT modern mode (GMT 6) was released in 2019.